images/capi/ansible/roles/containerd/tasks/main.yml

```diff
@@ -41,36 +54,18 @@
       - --no-overwrite-dir
   when: ansible_os_family != "Flatcar"
 
-- name: unpack containerd for Flatcar into /opt
-  unarchive:
-    remote_src: True
-    src: /tmp/containerd.tar.gz
-    dest: /opt
-    exclude:
-      - ./usr/local/bin/containerd
-      - ./usr/local/bin/containerd-stress
-      - ./usr/local/bin/critest
-      - ./usr/local/sbin/runc
-      - ./etc
-      - ./opt
-    extra_opts:
-      - --no-overwrite-dir
-      - --strip=3
-  when: ansible_os_family == "Flatcar"
+- name: Create a directory if it does not exist
+  file:
+    path: "{{ sysusr_prefix }}/bin"
+    state: directory
+    mode: 0755
 
-- name: unpack containerd for Flatcar into /etc
-  unarchive:
-    remote_src: True
-    src: /tmp/containerd.tar.gz
-    dest: /
-    exclude:
-      - ./etc/systemd/system/containerd.service
-      - ./etc/containerd/config.toml
-      - ./usr
-      - ./opt
-    extra_opts:
-      - --no-overwrite-dir
-  when: ansible_os_family == "Flatcar"
```

Comment:
does it need to maintain this `when`?

Reply:
Before this PR, we needed to maintain the `when`.
Now we are removing the conditions for cri-containerd tarballs, as suggested by Moshe.